supervisor/iptablesctrl/acls.go

```diff
@@ -64,7 +64,6 @@ func (i *Instance) chainRules(appChain string, netChain string, ip string) [][]s
 		i.appAckPacketIPTableContext,
 		i.appPacketIPTableSection,
 		"-s", ip,
-		"-p", "tcp",
```

Comment:
We make it protocol agnostic for chainRules but not for cgroupChainRules. Will that be dealt with later or did we forget to update that ?